src/providers/info.rs

```diff
@@ -129,33 +140,26 @@ impl InfoProvider for DefaultInfoProvider<'_> {
     }
 
     fn address_info(&mut self, address: &str) -> ApiResult<AddressInfo> {
```

Comment:
should we call this `contract_info` to be more explicit?

Reply:
will do :+1: